src/couchdb/couch_httpd_db.erl

```diff
@@ -1015,8 +1015,8 @@ db_attachment_req(#httpd{method=Method,mochi_req=MochiReq}=Req, Db, DocId, FileN
     _ ->
         [{"Location", absolute_uri(Req, "/" ++
             ?b2l(Db#db.name) ++ "/" ++
```

Comment:
Isn't the database name supposed to be composed only of a subset of ASCII characters?

```
% curl -X PUT http://localhost:5984/dbé                      
{"error":"illegal_database_name","reason":"Name: 'dbé'. Only lowercase characters (a-z), digits (0-9), and any of the characters _, $, (, ), +, -, and / are allowed. Must begin with a letter."}
```